lib/modules/platform/gerrit/__fixtures__/change-data.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
do we really need this big fixture? can't we inline a subset of this? (io perf on tests)

Reply:
i have removed it completely and replaced with partial<GerritChange>(...) objects